cmd/virt-handler/virt-handler.go

```diff
@@ -572,6 +578,38 @@ func (app *virtHandlerApp) shouldEnablePersistentReservation() {
 
 }
 
+// shouldEnableKSM will update the ksm of the node (if available) based on the kv configuration.
+// If the node labels match the selector terms, the ksm will be enabled.
+// Empty Selector will enable ksm for every node
+func (app *virtHandlerApp) shouldEnableKSM() {
+	node, err := app.virtCli.CoreV1().Nodes().Get(context.Background(), app.HostOverride, metav1.GetOptions{})
+	if err != nil {
+		log.DefaultLogger().Errorf("An error occurred while retrieving the node list: %s", err)
```

Comment:
Here the log message should be something like `An error occurred while retrieving the node object` since it's a `Get()` and not a `List()`?